### ISSUE TITLE
Rule to recommend toHaveBeenCalledOnceWith instead of toHaveBeenCalledTimes(1) and toHaveBeenCalledOnceWith()

### ISSUE THREAD

**Issue by rklec** [tlvince/eslint-plugin-jasmine#350]
**Is your feature request related to a problem? Please describe.**

Usually you can write code like this:
```js
expect(mySpy).toHaveBeenCalledTimes(1);
expect(mySpy).toHaveBeenCalledWith('foo');
```

**Describe the solution you'd like**
I want a warning that tells me I should use this:
```js
expect(mySpy).toHaveBeenCalledOnceWith('foo');
```

All these methods are part of standard jasmine: https://jasmine.github.io/api/edge/matchers.html

**Describe alternatives you've considered**
N/A

**Additional context**
This may technically similar to [`prefer-toHaveBeenCalledWith`](https://github.com/tlvince/eslint-plugin-jasmine/blob/master/docs/rules/prefer-toHaveBeenCalledWith.md).
